docs/content/drafts/UnderlineNav2.mdx

```diff
@@ -1,38 +1,129 @@
 ---
 title: UnderlineNav v2
+componentId: underline_nav_2
 status: Draft
 description: Use an underlined nav to allow tab like navigation with overflow behaviour in your UI.
+source: https://github.com/primer/react/tree/main/src/UnderlineNav2
+storybook: https://primer.style/react/storybook/?path=/story/layout-underlinenav
 ---
 
+```js
+import {UnderlineNav} from '@primer/react/drafts'
+```
+
 ## Examples
 
 ### Simple
 
 ```jsx live drafts
-<UnderlineNav label="simple nav">
+<UnderlineNav>
   <UnderlineNav.Item selected>Item 1</UnderlineNav.Item>
   <UnderlineNav.Item>Item 2</UnderlineNav.Item>
   <UnderlineNav.Item>Item 3</UnderlineNav.Item>
 </UnderlineNav>
 ```
 
+### Small variant
+
+```jsx live drafts
+<UnderlineNav variant="small">
+  <UnderlineNav.Item selected>Item 1</UnderlineNav.Item>
+  <UnderlineNav.Item>Item 2</UnderlineNav.Item>
+</UnderlineNav>
+```
+
 ### With icons
 
 ```jsx live drafts
-<UnderlineNav label="simple nav with icons">
-  <UnderlineNav.Item selected leadingIcon={EyeIcon}>
-    Item 1
+<UnderlineNav>
```

Comment:
Instead of repeating the lines here, I wanted to write it with react like [this](https://github.com/primer/react/blob/main/docs/content/ActionList.mdx#with-groups) but I am having a trouble to import UnderlineNav from drafts 😞 It is fine to import it on a different project like I tried to import to my primer prototype project but not sure what is going on here. Maybe we can pair tomorrow? @pksjce